tests/test_abort_file.py

```diff
@@ -38,30 +38,42 @@
 from tuna.machine import Machine
 from tuna.sql import DbCursor
 from tuna.tables import ConfigType
-
-
-def add_job():
-  find_configs = "SELECT count(*), tag FROM conv_config_tags WHERE tag='test_abort' GROUP BY tag"
-
-  del_q = "DELETE FROM conv_job WHERE reason = 'tuna_pytest'"
-  ins_q = "INSERT INTO conv_job(config, state, solver, valid, reason, session, fin_step) \
-        SELECT conv_config_tags.config, 'new', NULL, 1, 'tuna_pytest', 1, 'miopen_find_compile,miopen_find_eval' \
-        FROM conv_config_tags WHERE conv_config_tags.tag LIKE 'test_abort'"
-
-  print(ins_q)
-  with DbCursor() as cur:
-    cur.execute(find_configs)
-    res = cur.fetchall()
-    if len(res) == 0:
-      add_cfg = "{0}/../tuna/import_configs.py -f {0}/../utils/recurrent_cfgs/alexnet_4jobs.txt -t test_abort -C convolution".format(
-          this_path)
-      os.system(add_cfg)
-
-    cur.execute(del_q)
-    cur.execute(ins_q)
+from utils import CfgImportArgs, LdJobArgs
+from tuna.tables import DBTables
+from tuna.import_configs import import_cfgs
+from tuna.db_tables import connect_db
+from load_job import test_tag_name as tag_name_test, add_jobs
 
 
 def test_abort():
+  #import configs
+  session = 1
```

Comment:
Avoid hard coding session here. This will prevent the pytest from running stand-alone.